services/content-publishing/libs/common/src/utils/ipfs.client.ts

```diff
@@ -136,11 +126,10 @@ export class IpfsService {
     return response && response.data && JSON.stringify(response.data).indexOf(v0Cid) >= 0;
   }
 
-  private async ipfsHashBuffer(fileBuffer: Buffer): Promise<string> {
+  public async ipfsHashBuffer(fileBuffer: Buffer): Promise<string> {
     this.logger.debug(`Hashing file buffer with length: ${fileBuffer.length}`);
-    const hashed = await hasher.digest(fileBuffer);
-    const hash = create(hasher.code, hashed.bytes);
-    return base58btc.encode(hash.bytes);
+    const hash = hasher.digest(fileBuffer);
+    return base58btc.encode((await hash).bytes);
```

Comment:
nit: any reason why we don't `await` on line 131 instead of 132? Just looks weird this way...
```suggestion
    const hash = await hasher.digest(fileBuffer);
    return base58btc.encode(hash.bytes);
```